lib/Basic/Platform.cpp

```diff
@@ -195,21 +197,17 @@ StringRef swift::getPlatformNameForTriple(const llvm::Triple &triple) {
 }
 
 StringRef swift::getMajorArchitectureName(const llvm::Triple &Triple) {
-  if (Triple.isOSLinux()) {
-    switch(Triple.getSubArch()) {
-    default:
-      return Triple.getArchName();
-      break;
-    case llvm::Triple::SubArchType::ARMSubArch_v7:
-      return "armv7";
-      break;
-    case llvm::Triple::SubArchType::ARMSubArch_v6:
-      return "armv6";
-      break;
+    if (Triple.isOSLinux()) {
+        switch (Triple.getSubArch()) {
+            case llvm::Triple::SubArchType::ARMSubArch_v7:
```

Comment:
This still isn't quite right.  LLVM style is to align the `case`s with the `switch`.  This whole block just needs to be outdented.